doc/query.md

```diff
@@ -264,6 +264,23 @@ To query for documents where `age` is greater than twelve use the `$gt` conditio
 
 See below for supported operators (Selections -> Conditions).
 
+#### Modulo operation in queries
+
+Using the `$mod` operator in queries allows you to select documents based on the value of a field divided by a number yielding a specific remainder.
+
+To query for documents where `age` divided by 5 has a remainder of  4, do the following:
+
+```objc
+@{ @"age": @{ @"$mod": [ @5, @4 ] } }
+```
+
+A few things to keep in mind when using `$mod` are:
+
+- The array argument to the `$mod` operator must contain two number elements. The first element is the divisor and the second element is the remainder.
+- Division by zero is not allowed so the divisor cannot be zero.
+- The dividend (field value), divisor, and the remainder can be positive or negative.
+- The dividend, divisor, and the remainder can be represented as whole numbers or by using decimal notation.  However internally, prior to performing the modulo arithmetic operation, the dividend and divisor are rounded down to the nearest integer for positive values and rounded up to the nearest integer for negative values while the remainder is untouched.  So the query `@{ @"age": @{ @"$mod": [ @5.6, @4 ] } }` will provide the same result as the query `@{ @"age": @{ @"$mod": [ @5, @4 ] } }`.
+
```

Comment:
No longer necessary.  This has been updated and the remainder is now also truncated.